share/lib/python/neuron/rxd/node.py

```diff
@@ -646,13 +647,13 @@ def satisfies(self, condition):
         except:
             pass  # ignore the error and try Node3D specific conditions
 
-        if isinstance(condition, tuple) and len(condition) == 3:
+        if isinstance(condition, tuple) and len(condition) == 3: # Int replaced by floor to accept negative coordinates.
```

Comment:
We don't need to put this comment because (1) it's covered by the PR, and (2) it refers to code that no longer exists.